dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/ServletFilter.java

```diff
@@ -44,6 +44,7 @@ public class ServletFilter implements Filter {
 	
 	private static final String REST_HTTP_ROUTE_ATTR = "REST.HTTP.ROUTE";
 	
+	private static final String SPRING_BEST_MATCHING_PATTERN = "org.springframework.web.servlet.HandlerMapping.bestMatchingPattern";
```

Comment:
Don't want to have a runtime dependency on spring boot bundle/libs for this bundle. So using explicit constant over the one provided in the spring boot classes.